app/src/androidTest/java/net/skyscanner/backpack/docs/DocsRegistry.kt

```diff
@@ -131,6 +131,8 @@ object DocsRegistry {
     ViewScreenshot("Spinner - View - Default", "default"),
     ViewScreenshot("Spinner - View - Small", "small"),
     ComposeScreenshot("Spinner - Compose", "default"),
+    ViewScreenshot("Skeleton - View", "default"),
```

Comment:
tiny thing: could you move this into alphabetical order?

Reply:
make sense, will move it.